pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -203,6 +203,26 @@ func (r *ReconcileInfinispan) Reconcile(request reconcile.Request) (reconcile.Re
 		}
 	}
 
+	// Persistent volume claim for custom variables
+	if infinispan.Spec.Container.CustomLibrariesPVName != "" {
+		customPvc := &corev1.PersistentVolumeClaim{}
+		err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: infinispan.Namespace, Name: infinispan.Name + "-custom-libraries"}, customPvc)
+		if err != nil {
+			if errors.IsNotFound(err) {
```

Comment:
We have `LookupResource` func for this case

Reply:
yeah. Also I'm not sure if we want to keep pvc creation here or join it with the stateful set creation

Reply:
`LookupResource` doesn't fit very well here, it supposes that the resources is going to be created by another controller (i.e. service or secret). I would keep pvc creation tight to statetulset creation.

Reply:
Yeah, agreed